sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -694,6 +694,25 @@ Console.WriteLine($"Translation ({translation.Duration.Value.TotalSeconds}s):");
 Console.WriteLine(translation.Text);
 ```
 
+
+### Generate Speech with Text-to-Speech (TTS) models
+
+```C# Snippet:SpeechGeneration
+string text = "Hello World";
```

Comment:
I would remove this line of code if we want the code to look more concise.

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/11/commits/1ccebca8e08f53c52cfe952b5674f4272be5f705

Reply:
I should have been more clear. What I meant was you can directly set "Hello World" to speechOptions.Input

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/11/commits/97df7df2bcd0ed8913c577627cab60952e3bfc23